### PR TITLE
add wikitree images #126

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/wikitree/wikitree.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  try {
+    const data = await getData(
+      `https://cors-anywhere.herokuapp.com/https://api.wikitree.com/api.php?action=getProfile&key=${wikitreeId}`,
+      {'X-Requested-With': 'XMLHttpRequest'}
+    );
+    // console.log(data);
+    if (data[0] && data[0].profile.PhotoData && data[0].profile.PhotoData.url) {
+      return 'https://www.wikitree.com' + data[0].profile.PhotoData.url;
+    }
+  }catch (e) {
```
Comment (ogroppo) [inline src/wikitree/wikitree.js:13]: https://www.jetbrains.com/help/phpstorm/prettier.html - your prettifier should add a space `} catch`

File: src/wikitree/wikitree.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import getData from "../axios/getData";
+
+export default async function getWikitreeComImages(wikitreeId) {
```
Comment (ogroppo) [inline src/wikitree/wikitree.js:3]: I would call this function `getWikitreeImageUrl`, and rename the file accordingly

**Review by ogroppo — CHANGES_REQUESTED** [review]

File: src/wikitree/wikitree.js (hunk 1 above)
Comment (mshd) [reply src/wikitree/wikitree.js:13]: Ok I did it

File: src/wikitree/wikitree.js (hunk 2 above)
Comment (mshd) [reply src/wikitree/wikitree.js]: Ok

**Pull request merged**